src/core/mac/data_poll_sender.cpp

```diff
@@ -307,7 +307,7 @@ void DataPollSender::HandlePollTimeout(void)
     return;
 }
 
-void DataPollSender::CheckFramePending(Mac::RxFrame &aFrame)
+void DataPollSender::CheckFramePending(Mac::RxFrame &aFrame, bool aIsAck)
```

Comment:
For OT1.2, data poll is used to retrieve pending packets rather than check and retrieve data from parent. So the naming of this function does not really reflect its purpose here.